plugin/pkg/scheduler/algorithm/predicates/predicates_test.go

```diff
@@ -459,6 +459,16 @@ func TestPodFitsResources(t *testing.T) {
 			},
 		},
 		{
+			pod: newResourcePod(schedulercache.Resource{StorageOverlay: 18}),
+			nodeInfo: schedulercache.NewNodeInfo(
+				newResourcePod(schedulercache.Resource{MilliCPU: 2, Memory: 2, StorageOverlay: 5})),
+			fits: false,
+			test: "request exceeds allocatable",
+			reasons: []algorithm.PredicateFailureReason{
+				NewInsufficientResourceError(v1.ResourceStorageScratch, 18, 5, 20),
```

Comment:
I copied this from other test cases, but do not understand why `v1.ResourceStorageScratch` is used in test instead of `v1.ResourceStorageOverlay`?